mix.exs

```diff
@@ -10,26 +10,31 @@ defmodule PhoenixSlime.Mixfile do
       description: "Phoenix Template Engine for Slim-like templates",
       elixir: "~> 1.4",
       package: package(),
-      version: @version]
+      version: @version
+    ]
   end
 
   def application do
     [applications: [:phoenix, :slime]]
   end
 
   def deps do
-    [{:phoenix, "~> 1.3"},
-     {:phoenix_html, "~> 2.10"},
-     {:cowboy, "~> 1.0"},
-     {:slime, "~> 1.0"},
-     {:ex_doc, ">= 0.0.0", only: :dev},
-     {:earmark, ">= 0.0.0", only: :dev}]
+    [
+      {:phoenix, "~> 1.4"},
+      {:phoenix_html, "~> 2.10"},
+      {:jason, "~> 1.0", optional: true},
```

Comment:
Why do we need jason in dependencies?

Reply:
To get rid of the following Phoenix warning:

```
~/t/phoenix_slime ❯❯❯ mix test
Compiling 5 files (.ex)
Generated phoenix_slime app
warning: failed to load Poison for Phoenix JSON encoding
(module Poison is not available).

Ensure Poison exists in your deps in mix.exs,
and you have configured Phoenix to use it for JSON encoding by
verifying the following exists in your config/config.exs:

    config :phoenix, :json_library, Poison


  (phoenix) lib/phoenix.ex:40: Phoenix.start/2
  (kernel) application_master.erl:277: :application_master.start_it_old/4

...........

Finished in 1.1 seconds
11 tests, 0 failures
```

Reply:
Shouldn't it be only for test env then?

Reply:
I believe `optional: true` is the way to go there...

Reply:
I though about another issue with the current approach:

Now we have implicit configuration (`config :phoenix, :json_library, Jason`) that sets config value that is unrelated to `phoenix_slime` and may break someone else's code. 

Therefore I think it's better to move the dependency into `:test` env and config into `config/test.exs`.

@Rakoth @mschae What do you think?